Accessibility/VirtualContent/explainer.md

```diff
@@ -129,6 +129,72 @@ window.addEventListener("scroll", on_scroll_changed);
 13. Eventually, the backing server sends a content payload which also contains a signal that the end of the document has been reached. Script code removes the `aria-virtualcontent` attribute from the `main` element to indicate there is no more virtualized content.
 14. The next time the user attempts to navigate to the next heading, the AT searches for a virtual content container, finds none, and announces there are no further headings in the document.
 
+## Example 2: Tabular data
+
+This example illustrates one use of `aria-virtualcontent` for virtualized content in the inline direction. The document in this scenario is a table of bug reports with the following columns: report ID, report date, status, assigned to, title, fix date, fixed by. The last two columns have been virtualized out of the DOM based on the viewport width.
+
+```
+<html>
+<body>
+<table aria-virtualcontent="block-end inline-end" aria-colcount="7">
+  <thead>
+    <tr>
+      <th>ID</th>
+      <th>Status</th>
+      <th>Assigned To</th>
+      <th>Report Date</th>
+      <th>Title</th>
+ <!-- <th>Fix Date</th> - Virtualized -->
+ <!-- <th>Fixed By</th> - Virtualized -->
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>338</td>
+      <td>Closed</td>
+      <td>Alice</td>
+      <td>April 21</td>
+      <td>Widget freezes up when the network is slow</td>
+ <!-- <td>Fix Date</td> - Virtualized -->
+ <!-- <td>Fixed By</td> - Virtualized -->
+    </tr>
+    <tr>
+      <td>342</td>
+      <td>Fixed</td>
+      <td>Bob</td>
+      <td>April 22</td>
+      <td>Crash when shift-double-clicking the widget</td>
+ <!-- <td>Fix Date</td> - Virtualized -->
+ <!-- <td>Fixed By</td> - Virtualized -->
+    </tr>
+  </tbody>
+</table>
+</body>
+</html>
+```
+
+Backing script performing similar functions as Example 1's virtualcontent.js is assumed to be present.
+
+### Walkthrough
+
+An AT navigating the above content by table cells might result in the following flow:
+1. The AT stops on and reads out each of the five realized cells in the header row.
+2. Upon reaching the end of the row, the AT discovers that the table has virtualized content in the inline-end direction and scrolls the document in that.
```

Comment:
Seems like this is missing the word "direction" at the end of the sentence

Reply:
Indeed, thanks.